src/feature/MovieDetails/MovieDeatilsPage/Cast/styled.js

```diff
@@ -2,75 +2,93 @@ import styled from "styled-components";
 import { Link } from "react-router-dom";
 
 export const Wrapper = styled.div`
-  width: 1368px;
-  width: 100%;
-  padding: 0;
-  margin: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(208px, 1fr));
-  grid-gap: 24px;
-@media(max-width: ${({ theme }) => theme.breakpoints.large}px) {
-    grid-template-columns: repeat(4, 1fr);
-}
-@media(max-width: ${({ theme }) => theme.breakpoints.mobileMax}px) {
-    grid-template-columns: repeat(3, 1fr);
-}
-@media(max-width: ${({ theme }) => theme.breakpoints.mobileMin}px) {
-    grid-template-columns: repeat(2, 1fr);
-    margin: 12px auto 0;
-}
+    margin: 52px auto 40px;
+    padding: 0 16px;
```

Comment:
This padding, I don't think we need it, when you remove it it makes 4 tiles to render in row not 3. Is this padding important?